scripts/emailings/reset-invalid-emails.php

```diff
@@ -67,38 +68,78 @@
 
 $db->begin();
 
-
-// TODO Loop on the entry file to get the 100 first entries
-
-$groupofemails = array();
-
-
-	// For each groupofemail, we update tables to set email field to empty
-	if ($type == 'all' || $type == 'thirdparty')
+$myfile = fopen($id, "r") or die("Unable to open file\n");
+$groupofemails=array();
+for ($i=0; $i < MAXEMAILS; $i++)
```

Comment:
This part of code can be removed here. It has been moved later inside the loop 
`while ($tmp!=null)`
So it is a rest of the old PR i think to remove.